doc/source/tutorial/stats/continuous_gennorm.rst

```diff
@@ -0,0 +1,58 @@
+
+.. _continuous-gennorm:
+
+Generalized Normal Distribution
+===============================
+
+This distribution is also known as the exponential power distribution. It has
+a single shape parameter :math:`\beta>0`. It reduces to a number of common
+distributions.
+
+Functions
+---------
+
+.. math::
+    :nowrap:
+
+    \begin{eqnarray*} f\left(x\right) & = &\frac{\beta}{2\Gamma(1/\beta)} e^{-\left|x\right|^{\beta}} \end{eqnarray*}
```

Comment:
Here and elsewhere: the convention for specifying the funtion arguments is to include the parameter, separated from x with a semicolon.  E.g. `f\left(x; \beta \right) & = & ...`.